client/src/App.js

```diff
@@ -1,21 +1,18 @@
 import React, { Component } from "react";
 import { Route, Link } from "react-router-dom";
+import SplashPage from "./components/SplashPage/SplashPage";
 import "./App.css";
 
 class App extends Component {
   splash = () => {
-    return (
-      <div className='splash'>
-        <h1>Welcome to CoTrip</h1>
-      </div>
-    );
+    return <SplashPage />;
   };
   render() {
     return (
       <div className='App'>
-        <nav>
+        {/* <nav>
           <Link to="/">Splash</Link>
-        </nav>
+        </nav> */}
         <main>
           <Route path="/" exact render={this.splash}></Route>
```

Comment:
This isn't really how React Router's `render` work. And, since you're just rendering a single component, could you use the `component` property for now?